cherrypy/test/test_httputil.py

```diff
@@ -2,31 +2,50 @@
 import pytest
 
 from cherrypy.lib import httputil
+from six.moves import http_client
```

Comment:
Please note that the imports order must be the following: stdlib, then third-party and finally our project imports